doc/adr/0001-use-micro-frontends.md

```diff
@@ -0,0 +1,31 @@
+# 1. Use micro frontends
+
+Date: 2020-03-25
+
+## Status
+
+Proposed
+
+## Context
+
+We have designed the API layer (known as the backend) to be a custom set of services. This allows the user to tailor it
+to their own requirements, and introduce custom code as needed.
+
+We are currently planning to make the presentation layer (known as the frontend) a monolith. That is, a single service
```

Comment:
Does the language here make sense, for the fresh reader the "we are currently planning" in an ADR that makes a decision to do something else seems less clear. Perhaps "The original plan was to..."

Reply:
I noticed too. ADRs are a log like events, which suggests past tense.